www/index.html

```diff
@@ -62,6 +62,9 @@
             },
             onDeviceReady: function() {
                 document.addEventListener("backbutton", onBackPressed, false);
+                if (cordova.platformId.toLowerCase() === 'ios') {
```

Comment:
That is totally not what we want.
We want it not to overlay with main UI, but not hidden.